src/index.ts

```diff
@@ -75,6 +78,9 @@ const parseArgs = (args: string[]) => {
       case "--avoid-single-file":
         cliConfig.avoidSingleFile = true;
         break;
+      case "--ignore":
+        cliConfig.ignore = [...(cliConfig.ignore ?? []), args.shift() ?? ""];
+        break;
```

Comment:
```suggestion
      case "--ignore":
        const nextOptionIdx = args.findIndex(x => x.startsWith("-"));

        cliConfig.ignore = [...(cliConfig.ignore ?? []), ...(args.splice(0, nextOptionIdx) ?? [])];
        break;
```
https://github.com/benawad/destiny/issues/91#issuecomment-593820780

So it can support multiple paths / globs: `destiny src --ignore src/pages *.js -w`